addons/OPT/REVIVE/fn_clientInitEH.sqf

```diff
@@ -57,25 +57,24 @@ DFUNC(isUnconscious) =
 
 //var nach Respwan zurück setzen
 ["Respawn", {
-
-    {
-		player setVariable ["OPT_isUnconscious", 0, true];
-		player setVariable ["OPT_isStabilized", 0, true];
-		player setVariable ["OPT_isDragged", 0, true];
-		GVAR(OPT_isDragging) = false;
-		player allowDamage true;
-		1 enableChannel true;
-		1 fadeSound 1;
-		OPT_GELDZEIT_earplugsInUse = 1;
-		OPT_REVIVE_unconsciousHandler = nil;
-		OPT_REVIVE_respawnedHandler = nil;
-		player setVariable ["tf_unable_to_use_radio", false];
-
-		//Schaden Freigeben
-		player allowDamage true;
-		 
-    } call CFUNC(execNextFrame);
-
+	params ["_data", "_args"];
+	_data params ["_newPlayer", "_oldPlayer"];
+	_oldPlayer removeEventHandler ["HandleDamage", GVAR(PLAYER_HANDLE_DAMAGE_EH_ID)];
+
+	GVAR(PLAYER_HANDLE_DAMAGE_EH_ID) = _newPlayer addEventHandler ["HandleDamage", FUNC(playerHandleDamage)];
```

Comment:
Da sollte am Besten ein Kommentar rein, der erklärt warum es nötig ist den EH erst zu entfernen und dann neu hinzuzufügen

Reply:
Das weiß ich selber noch nicht zu 100%. 
Joko meinte zu mir dass sich das _player Objekt beim Respawn ändert und es deßwegen notwendig ist.
Schreibe ich dazu.

Reply:
Ja das player Objekt ändert sich. Nur hatten wir in unseren Tests ja gesehen, dass der EH tdem persistent zu sein schien :shrug: